packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -597,7 +596,7 @@ export class TextField {
                   ["readonly"]: readonly,
                   ["truncate-value"]: truncateValue,
                 }}
-                placeholder={placeholderText}
+                placeholder={~~placeholder ? placeholder : ""}
```

Comment:
@GCHQ-Developer-530 I used bitwise comparrison instead of boolean, if that is ok? It's slightly faster in terms of performance, even though the amount of passes through this slice of code is not that much.